lib/beaker/dsl/install_utils.rb

```diff
@@ -1,17 +1,12 @@
-[ 'foss', 'puppet', 'ezbake', 'module' ].each do |lib|
-    require "beaker/dsl/install_utils/#{lib}_utils"
-end
 require "beaker/dsl/install_utils/pe_defaults"
+require 'beaker-puppet'
 
 module Beaker
   module DSL
     # Collection of installation methods and support
     module InstallUtils
-      include DSL::InstallUtils::PuppetUtils
       include DSL::InstallUtils::PEDefaults
```

Comment:
Not a blocker for anything, but should this be moved to beaker-pe? Is that ticketed anywhere?